main.nf

```diff
@@ -776,7 +780,7 @@ if( !params.classifier ){
 		"""
 		export HOME="\${PWD}/HOME"
 
-		if [ \"${params.unite}\" = \"true\" ]; then
+		if [ ${params.taxon_reference} = "unite" ]; then
```

Comment:
So one of the two (old solution or new one) will not work.
Typically `"` needs to be escaped with `\` in a block of string starting & ending with `"""`, such as this block. But if this code works, I wont complain.

Reply:
Hm, I understand what you mean, and for some reason my code worked, with or without escaping. But the old solution is probably more solid, so I'll go with escaping - but also with quotes for the parameter. If that's ok? (Fixing it now.)

Reply:
Not sure if the parameter requires quotation mark now, I'd think it doesn't. Before it did, because true was also in quotation marks.